Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitTestHelper.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+///Provides helper functions to test the SEPA Direct Debit flow in sandbox.
```

Comment:
```suggestion
/// Provides helper functions to test the SEPA Direct Debit flow in sandbox.
```

Reply:
Should we specify that this file + it's methos aren't under version control? Or should it adhere to SemVer?

Reply:
Similar / follow-up question - do we think this is worth calling out in the CHANGELOG?

Reply:
🤔 Yeah it may be worth calling that out just incase we need to make changes that we aren't thinking of currently. Not sure what that would be but better safe than getting ourselves stuck. We can also add a changelog entry for this!

This is also currently targeting the `master` branch. We probably want this to go into `5.x` and merge that into `master` eventually correct?

Reply:
Braintree SEPA is not in beta, right? If not - then I think you're right we'll want to make the change on `5.x` and then be sure to regularly be merging `5.x` into master.

This way - we can cut a `5.x` release with this SEPA update (for example) on a stable v5 release.

Reply:
Cool - I thought it'd be easy-ish to change the base branch but looks like it's gonna be weird, so gonna close this PR and open a new one with these changes pointing to 5.x